telemetry/specs/INT.mdk

```diff
@@ -336,6 +440,10 @@ https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-d
 # INT Headers
 
 This section specifies the format and location of INT Headers.
+INT Headers and their locations are relevant for
+INT-MX, INT-MD and INT-CLONE/PROBE-MD modes
```

Comment:
INT-CONE/PROBE-MD is confusing. Let's explicitly state that the INT headers and their locations are relevant for INT-MX, INT-MD, INT-CLONE-MD, and INT-PROBE-MD modes.

Reply:
Sure, done.